src/tests/ftest/nvme/io_verification.py

```diff
@@ -77,8 +77,8 @@ def test_nvme_io_verification(self):
                     self.ior_cmd.block_size.update(32000)
                 else:
                     self.ior_cmd.block_size.update(self.ior_block_size)
-                self.ior_cmd.set_daos_params(self.server_group, self.pool)
-                self.job_manager.job.dfs_cont.update(self.label_generator.get_label('cont'))
+                self.ior_cmd.set_daos_params(
+                    self.server_group, self.pool, self.label_generator.get_label('cont'))
```

Comment:
Should we be consistent and use `TestContainer`?

Reply:
I was trying to stick with previous behavior as much as possible, which is that IOR creates the container, not the test. I.e. if you give IOR a container label, and the container doesn't exist, IOR creates it.
The problem before was that IOR utils was generating a UUID, but:
1. We should be using labels, not UUID.
2. The underlying DAOS API ignores a user-supplied UUID on container create. So the UUID that the test passed wasn't the UUID of the container created